### PR TITLE
Removed scalar and index fields

### LINKED ISSUE #70: Delete scalar indices from seq tables
For PandA tables, we store them in waveform records, with one for each column, We also have an `index` record, and a scalar record per column that allows you to see a single row of the table. This was conceived before Phoebus had PVA NTTable support, and never worked properly. In fact it makes it fail if you have an empty table. I propose we delete the index and scalar records from tables.

@dmgav you aren't using this feature are you?

### PR DESCRIPTION
**Pull request by evalott100**
Closes #70

### CODE REVIEW

**Comment by evalott100** [comment]
SEQ table before:
image
and after
image

File: src/pandablocks_ioc/ioc.py [hunk 1]
```diff
@@ -893,11 +893,13 @@ def _make_pos_out(
...
-            positions_record_name + ":NAME",
-            VAL=record_name,
-            DESC="Table of configured positional outputs",
-        ),
+        (
+            builder.records.stringin(
+                positions_record_name + ":NAME",
+                VAL=record_name,
+                DESC="Table of configured positional outputs",
+            ),
```
Comment (coretl) [inline src/pandablocks_ioc/ioc.py:901]: Not sure why there was a trailing comma here...
```suggestion
            )
```

**Review by coretl — APPROVED** [review]

**Review by AlexanderWells-diamond — CHANGES_REQUESTED** [review]
Unless I've misunderstood something, there are still references left to both SCALAR and INDEX records in the codebase. A short list includes:
- `TableUpdater.table_scalar_records`
- In test_tables.py `table_updater` contains `updater.index_record` mocking. 

Please check through all occurrences of the word "scalar" and "index" to ensure all references have been cleaned up.

**Comment by evalott100** [comment]
> Unless I've misunderstood something, there are still references left to both SCALAR and INDEX records in the codebase. A short list includes:
> 
>     * `TableUpdater.table_scalar_records`
> 
>     * In test_tables.py `table_updater` contains `updater.index_record` mocking.
> 
> 
> Please check through all occurrences of the word "scalar" and "index" to ensure all references have been cleaned up.

Apologies accidentally didn't push some work before I asked Tom for review.

File: src/pandablocks_ioc/_tables.py [hunk 2]
```diff
@@ -84,7 +82,6 @@ class TableUpdater:
     # Order is exactly that which PandA sent.
     table_fields_records: typing.OrderedDict[str, TableFieldRecordContainer]
     # Collection of the records that comprise the SCALAR records for each field
```
Comment (AlexanderWells-diamond) [inline src/pandablocks_ioc/_tables.py:84]: This comment should be deleted

**Review by AlexanderWells-diamond — APPROVED** [review]
Delete the one extra comment then this looks good to go.

**Pull request merged**